### DISCUSSION TITLE
希望优化“评论区IP属地显示”功能的排版

### DISCUSSION THREAD

**Discussion by NickerGrylls** [the1812/Bilibili-Evolved#4506] [功能建议 / Ideas]
个人认为当前该功能显示的时间戳和IP属地之间距离过小

> ![image](https://github.com/the1812/Bilibili-Evolved/assets/120347871/9a6abcf4-a019-4032-9af7-0e5105ef00d1)

相比之下，我之前用过的[另一个脚本](https://greasyfork.org/zh-CN/scripts/448434-b%E7%AB%99%E8%AF%84%E8%AE%BA%E5%8C%BA%E5%BC%80%E7%9B%92)看起来就会更加明晰，也和后面点赞评论之间的距离一致，外观上更加统一

> ![image](https://github.com/the1812/Bilibili-Evolved/assets/120347871/e9e3ab46-f92f-46aa-aa2a-55bf51e1c2ee)

虽是个细节，但还是希望能够优化

**Comment by the1812** [comment]
@LightQuanta

**Comment by LightQuanta** [comment]
收到，等我下周有空了改一下

**Reply by ChaXxl** [reply]
Hello，有空了吗，可以改一下吗

**Reply by LightQuanta** [reply]
最近比较忙，在整一堆别的项目（
~~年后我看看吧，2月内应该能整出来~~

呃 我这突然有急事，2月没空了，3月再说吧

**Comment by LightQuanta** [comment]
#4689
